utah-geological-survey.csl

```diff
@@ -135,9 +135,9 @@
           <group prefix=", ">
             <text term="in" font-style="italic"/>
             <names variable="editor translator" prefix=" " suffix="," delimiter=", ">
-              <name and="text" sort-separator=", " initialize-with="." name-as-sort-order="all"/>
+              <name and="text" sort-separator=", " initialize-with=".," name-as-sort-order="all"/>
               <et-al term="and others"/>
-              <label form="short" prefix=" "/>
+              <label form="long" prefix=" "/>
```

Comment:
(it's not wrong, but typically we just delete `form="long"` since it's the default)